examples/segmentation_camera/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+cmake_minimum_required(VERSION 3.10.2 FATAL_ERROR)
```

Comment:
Do you main to add support for windows in the CMakelists and example ? [here a reference](https://github.com/ignitionrobotics/ign-rendering/pull/291)

Reply:
No, I don't know how to add it :D, i just copied that from the examples